src/main/java/org/jboss/ws/plugins/tools/AbstractWsConsumeMojo.java

```diff
@@ -147,10 +147,19 @@ public void execute() throws MojoExecutionException
          params.setWsdlLocation(wsdlLocation);
          params.setEncoding(encoding);
          params.setArgLine(argLine);
-         params.setFork(fork);
-         
-         File manifestOnlyJar = createJar(getClasspathElements(), "");
-         params.setManifestOnlyJar(manifestOnlyJar);
+         if (fork || Util.getJVMMajorVersion() > 8) {
+        	 params.setFork(true);
+             File manifestOnlyJar = createJar(getClasspathElements(), "");
+             params.setManifestOnlyJar(manifestOnlyJar);
+             if (verbose)
+             {
+                log.info("Additional plugin classpath:");
+                for (File f : getRequiredPluginDependencyPaths())
+                {
+                   log.info(" " + f.getAbsolutePath());
```

Comment:
Info?

Reply:
yes, that's only when the "verbose" flag is on